en-US/firefox/best-browser.lang

```diff
@@ -0,0 +1,117 @@
+## NOTE: Demo page at https://bedrock-demo-achurchwell.oregon-b.moz.works/firefox/best-browser/
+## URL: https://bedrock-demo-achurchwell.oregon-b.moz.works/%LOCALE%/firefox/best-browser/
+
+;Find your best browser for speed, privacy and security.
```

Comment:
Can add a comment that this is an HTML page title - usually translation should be short.